deriva/core/deriva_binding.py

```diff
@@ -177,6 +268,7 @@ def get(self, path, headers=DEFAULT_HEADERS, raise_not_modified=False, stream=Fa
         if headers is None:
             headers = {}
         url, headers, prev_response = self._pre_get(path, headers)
+        print('Running GET %s %s' % (path, headers))
```

Comment:
Is this `print` statement intentional or just left over from development testing? I notice this module isn't configured to use `logging`, maybe we should add it and change this to a debug logging statement?